sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -833,6 +833,46 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OverrideTenantExternalUserExpirationPolicy
+
+Choose whether to override the external user expiration policy on this site
+
+PARAMVALUE: None | False | True
+
+- None - Respect the organization-level policy for external user expiration
+- False - Respect the organization-level policy for external user expiration
+- True - Override the organization-level policy for external user expiration (can be more or less restrictive)
+
+```yaml
+Type: Boolean
+Parameter Sets: ParamSet1
+Aliases:
+Applicable: SharePoint Online
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ExternalUserExpirationInDays
+
+Specifies all external user expiration which will expire after the set number of days. Only applies if OverrideTenantExternalUserExpirationPolicy is set to true.
+
+To remove the expiration requirement, set the value to zero (0).
```

Comment:
Change to:

"...set the value to 0 (zero).".  This is to explain that you're setting the value to the numeral, otherwise it reads as if you're setting the value to the textual representation of the numeral.